multi-tenancy/README.md

```diff
@@ -0,0 +1,134 @@
+---
+creation_date: 2024-02-08
+issues:
+- https://github.com/giantswarm/roadmap/issues/2771
+owners:
+- https://github.com/orgs/giantswarm/teams/team-atlas
+state: review
+summary: How GiantSwarm manages multi-tenancy to isolate data by tenants (i.e. metrics, logs, traces).
+---
+
+# Multi-tenancy proposal
```

Comment:
In our public docs we already use `multi-tenancy` to name the features and structure around Organizations. Since this is not about that, could you consider changing the title to reflect that they are not the same topic? 

```suggestion
# Multi-tenancy on monitoring
```

same goes for the folder name and other generic references to "multi-tenancy". wdyt?